extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/uriExtractors.scala

```diff
@@ -33,7 +33,11 @@ object URIExtractor {
  */
 class StringURIExtractor extends URIExtractor {
   override def apply(v1: AnyRef): Seq[Uri] = {
-    Seq(Uri(v1.asInstanceOf[String]))
+    v1 match {
+      case str: String => Seq(Uri(str))
+      case Some(str: String) => Seq(Uri(str))
```

Comment:
```suggestion
      case uriPath: String => Seq(Uri(uriPath))
      case Some(uriPath: String) => Seq(Uri(uriPath))
```
Avoid using the data type name as the variable name.

Reply:
Done